src/main/java/de/muenchen/dave/services/ZaehlstelleIndexService.java

```diff
@@ -642,6 +643,24 @@ private void updateStatusOfZaehlung(final Zaehlung zaehlung, final Status newSta
         this.messageService.saveUpdateMessageForZaehlungStatus(zaehlung.getId(), updateStatusDTO);
     }
 
+    public List<LadeZaehlstelleWithUnreadMessageDTO> readZaehlstellenWithUnreadMessages() {
+        final List<LadeZaehlstelleWithUnreadMessageDTO> zaehlstellen = readZaehlstellenWithUnreadMessages(Participant.MOBILITAETSREFERAT.getParticipantId());
+        zaehlstellen.forEach(zaehlstelle -> zaehlstelle.setZaehlungen(zaehlstelle.getZaehlungen()
+                .stream()
+                .filter(LadeZaehlungWithUnreadMessageDTO::getUnreadMessagesMobilitaetsreferat)
+                .collect(Collectors.toList())));
+        return zaehlstellen;
+    }
+
+    public List<LadeZaehlstelleWithUnreadMessageDTO> readZaehlstellenWithUnreadMessagesExternal() {
+        final List<LadeZaehlstelleWithUnreadMessageDTO> zaehlstellen = readZaehlstellenWithUnreadMessages(Participant.DIENSTLEISTER.getParticipantId());
+        zaehlstellen.forEach(zaehlstelle -> zaehlstelle.setZaehlungen(zaehlstelle.getZaehlungen()
```

Comment:
Innerhalb der ForEach-Funktion wegen Übersichtlichkeit die gefilterten Zählungen in eigener final Variablen vorhalten und dann dem Setter der Zählstelle übergeben.